### DISCUSSION TITLE
What does the "k" factor mean?

### DISCUSSION THREAD

**Discussion by deuko** [Chia-Network/chia-blockchain#2187] [Q&A] [answered]
Hello,

sorry if this was already answered, but I couldn't find anything in the documents or via the search.
What does the "k" factor indicate when plotting?

Is k=25 "worse" than k=32?
Do I have any disadvantage when plotting more k=25 small plots instead of bigger plots?

Thank you in advance

**Comment by dorkmo** [comment, marked as answer]
32 is the minimum
